lib/screens/theme_screen/widgets/skip_and_next_row.dart

```diff
@@ -21,20 +22,24 @@ class SkipAndNextRow extends StatelessWidget {
             width: 70,
             height: 50,
             child: ElevatedButton(
-              onPressed: () {},
+              onPressed: () {
+                  Navigator.of(context).push(
+                  MaterialPageRoute(builder: (context) => LoginScreen()));
```

Comment:
Why pushing to LoginScreen?

Reply:
Fixed with MainScreen() as main screen was not in my fork at that time